### ISSUE TITLE
Eager Op BUG汇总

### ISSUE THREAD

**Issue by BBuf** [Oneflow-Inc/oneflow#6003]
## Eager Op BUG汇总

这些BUG大多数都需要去做自动测试才能发现，这里记录一些目前发现但还没来得及解决的BUG。之前已经解决了数十个自动测试框架发现的Op BUG，但目前还有100+个API没有自动测试，所以也需要同时推进自动测试来发现更多OP中的BUG。

如果发现了其它BUG不能立即解决也可以更新在这个issue里。

| 接口名                                 | BUG类型            | 解决进度                                                  |
| -------------------------------------- | ------------------ | --------------------------------------------------------- |
| oneflow.std                            | 和Pytorch对不齐    |                                                           |
| oneflow.nn.CrossEntropyLoss            |   当输入是三维，shape和Pytorch对不齐                 | https://github.com/Oneflow-Inc/oneflow/pull/5799 正在解决 |
| oneflow.softsign                       | Pytorch反向存在BUG | 反馈给了Pytorch，他已经修了，但需要等待下一个版本         |
| consistent_tensor `x[0][0].to("cuda")` | 来自PR 5902的标记  |                                                           |
| 0-dim tensor的optimize有bug            | 来自PR 5552的标记  |                                                           |
| oneflow.min，Tensor.min                | 没对齐接口         |                                                           |
| oneflow.max，Tensor.max                | 没对齐接口         |                                                           |
| oneflow.sub，oneflow.div               | 没对齐接口         |                                                           |
|                                        |                    |                                                           |

**Comment by BBuf** [comment]
where op bug复现代码：

`
import oneflow as flow

x = flow.Tensor([[-0.4620,  0.3139], [ 0.3898, -0.7197], [ 0.0478, -0.1657]])
print(flow.where(x > 0, x, 0.))
`
